src/api/.rubocop_todo.yml

```diff
@@ -1,12 +1,12 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config --auto-gen-only-exclude --exclude-limit 100`
-# on 2024-07-15 11:49:40 UTC using RuboCop version 1.65.0.
+# on 2024-07-15 16:14:45 UTC using RuboCop version 1.65.0.
```

Comment:
If we only update the `Offense count` of some cops, we don't need to update this file.